containers/Convert/Convert.tsx

```diff
@@ -3,16 +3,19 @@ import axios from 'axios';
 import { ethers } from 'ethers';
 import { useRecoilValue } from 'recoil';
 import { useCallback } from 'react';
-
-import erc20Abi from 'lib/abis/ERC20.json';
+//@ts-ignore
+import getFormatedSwapData from '@jaredborders/synthswap-utils';
```

Comment:
Should we find a way to publish this from Kwenta? @JaredBorders

Reply:
We can. Will put on todo list for tmr

Reply:
Also could be good to add the types or convert the code to ts, maybe a nice devdao task

Reply:
Latest version of Synthswap package no longer uses that

Reply:
1.0.3